1st-Assignment/1st-Assignment/SignupMakeNameController.swift

```diff
@@ -9,13 +9,28 @@ import UIKit
 
 class SignupMakeNameController: UIViewController {
 
+    @IBOutlet weak var signupName: UITextField!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        self.navigationController?.navigationBar.topItem?.title = ""
+        
+        self.navigationController?.navigationBar.backIndicatorImage = UIImage(named: "icn_back_24")
+        self.navigationController?.navigationBar.backIndicatorTransitionMaskImage = UIImage(named: "icn_back_24")  
```

Comment:
```suggestion
        navigationController?.navigationBar.topItem?.title = ""
        
        navigationController?.navigationBar.backIndicatorImage = UIImage(named: "icn_back_24")
        navigationController?.navigationBar.backIndicatorTransitionMaskImage = UIImage(named: "icn_back_24")  
```

`self `키워드의 경우, 코드의 간결함을 위해서 지양하는 것이 좋습니다. 
반드시 써야 하는 경우가 있는데, 클로저 안에서 사용하거나 / 클래스 내의 변수와 함수의 파라미터로 받은 변수의 이름이 동일할 경우 등 명시적으로 어떤 것을 가리키는 지 확실하게 필요할 때 쓰는 것이 좋습니당 😇

Reply:
와 알겠습니다!!!!!!알려주셔서 감사해요💗,,